projects/packages/schema/tests/php/integration/Integration_Serialization_Test.php

```diff
@@ -186,7 +186,7 @@ public function test_serialization_assoc_value() {
 
 		// Test in both directions
 		$this->assertSame( $expect_schema_to_be, $schema->schema() );
-		$this->assertSame( $expect_schema_to_be, json_decode( wp_json_encode( $schema ), true ) );
+		$this->assertSame( $expect_schema_to_be, json_decode( wp_json_encode( $schema, JSON_UNESCAPED_SLASHES ), true ) );
```

Comment:
Another encode-decode that could use `JSON_UNESCAPED_UNICODE`.

```suggestion
		$this->assertSame( $expect_schema_to_be, json_decode( wp_json_encode( $schema, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE ), true ) );
```

Reply:
24d018fdee